docs/5.x/installation.md

```diff
@@ -216,15 +207,43 @@ Tue Apr 10 14:01:09 UTC	Executing install phase "/" finished in 5 minutes
 
 ```
 
-The following CLI flags are useful to manage the install operation:
+To abort an installation, press Ctrl+C two times in a row in the terminal.
+If the operation is aborted, the partial cluster state will be automatically removed.
 
+Aborting an agent (and not the installer process), will only prevent this node from joining.
+
+!!! warning "Aborting an agent":
+    Aborting an agent might result in installation failure.
+    If the agent was aborted to correct a configuration error, it can be restarted once the error
+    has been fixed.
+
+
+Installer processes (that includes the main installer process and all agent processes that join additional nodes to the cluster)
+execute inside a systemd service so the operation will continue in background even if the terminal session has timed out.
+
+In order to reconnect to the installer, issue `./gravity resume`.
+
+!!! warning "Installer state directory":
+    It is important to run install commands from the directory with the original gravity binary.
```

Comment:
It might be a good idea to try and eliminate this limitation in the future (or atleast have a ticket open to do so). I could see users missing this concept in the docs and bumping into this.

Reply:
What do you see here as limiting?
For install to work, it needs to know where to find original installer data (extracted from the tarball), so running `./gravity install` makes sense.
Then, it needs to look up state just like `git` does - it defaults to a local sub-directory to the directory from where the executable is invoked by default. It does not mean it needs to always use the same binary - it can be the one already installed in `/usr/bin`, but it needs to be the same directory with the install files.